src/xopen/__init__.py

```diff
@@ -732,28 +777,36 @@ def xopen(
     (parallel gzip) subprocess if possible. See PipedGzipWriter and PipedGzipReader.
 
     When threads = 0, no subprocess is used.
+
+    encoding, errors and newline are used when opening in text mode. The parameters
+    have the same meaning as in the built-in open function.
```

Comment:
... except that the default text encoding is UTF-8 on all platforms, not locale.getpreferredencoding().

Reply:
Well-spotted. I should change that.